### DISCUSSION TITLE
修改主题后非常不协调，建议优化一下

### DISCUSSION THREAD

**Discussion by kunyyds666** [1943time/inkdown#82] [Ideas]
![image](https://github.com/1943time/bluestone/assets/93746366/4dae40b7-f9f0-429f-a075-463142708ac9)

**Comment by 1943time** [comment]
顶部属于系统风格，electron更改顶部风格会把整个系统改为dark模式，影响其他软件，如果你需要顶部变为暗色，可以再window系统里设置为暗黑模式。
